### PR TITLE
Cleaned up OpenGL recyclebins

### PR DESCRIPTION
**Pull request by marcusnaslund**
Sometimes, rock really is your friend.

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Peer review @emilwestergren

**Comment by emilwestergren** [comment]
Nice

File: source/draw/gpu/opengl/OpenGLContext.ooc [hunk 1]
```diff
@@ -117,20 +99,14 @@ OpenGLContext: class extends GpuContext {
...
-		match (image) {
-			case (i: OpenGLMonochrome) => this _recycleBinMonochrome add(i)
-			case (i: OpenGLRgb) => this _recycleBinRgb add(i)
-			case (i: OpenGLRgba) => this _recycleBinRgba add(i)
-			case (i: OpenGLUv) => this _recycleBinUv add(i)
-			case => Debug error("Unknown format in recycle add()")
-		}
+		this _recycleBin add(image)
 	}
 	_searchImageBin: func (type: GpuImageType, size: IntVector2D) -> GpuImage {
```
Comment (emilwestergren) [inline source/draw/gpu/opengl/OpenGLContext.ooc]: Can you look into removing `GpuImageType` completely?

Comment (naslundx) [inline source/draw/gpu/opengl/OpenGLContext.ooc]: Sure. Create an issue and I'll look into it later.

Comment (marcusnaslund) [inline source/draw/gpu/opengl/OpenGLContext.ooc]: Sure. Create an issue and I'll look into it later.

**Pull request merged**